src/components/BooksQuery.tsx

```diff
@@ -53,13 +52,14 @@ const BooksQuery = () => {
         <button onClick={submit}>FIND</button>
       </div>
       <div className="books">
-        {list.map((book: book) => (
+        {list.books.map((book: BookObject) => (
```

Comment:
TypeScript can infer this type implicitly. there's no need to add it here explicitly